src/Slider.jsx

```diff
@@ -337,16 +337,20 @@ class Slider extends React.Component {
     const lowerOffset = this.calcOffset(lowerBound);
 
     const handleClassName = prefixCls + '-handle';
+
+    const upperClassName = handleClassName + ' ' + handleClassName + '-upper';
+    const lowerClassName = handleClassName + ' ' + handleClassName + '-lower';
+
     const isNoTip = (step === null) || (tipFormatter === null);
 
-    const upper = cloneElement(customHandle, { className: handleClassName,
-        noTip: isNoTip, tipTransitionName: tipTransitionName, tipFormatter: tipFormatter,
+    const upper = cloneElement(customHandle, { className: upperClassName,
+        noTip: isNoTip, tipTransitionName: tipTransitionName, tipFormatter: tipFormatter, upperHandle: true, lowerHandle: false,
```

Comment:
`className` is enough, just remove `upperHandle` and `lowerHandle` to simplify code.